Remarkable-Donkeys/bangazon
# Tb customers

# Steps To Test #
1. Pull TestingBranch from GitHub
2. Start running your server
3. Using Postman test POST, GET, and PUT operations on Customers Table.  
*Ensure that you can GET a single customer as well as all the customers.
4.Using Postman use POST to create a department.
5.Using Postman test POST, GET and PUT operations on Employee Table. 
*Ensure that you can GET a single employee as well as all employees. 

Ticket #1 
Ticket #6 

### Description of what should happen when steps are followed ###
If tests pass you should be able to POST, GET, and PUT a Customer and an Employee using Postman.  There should be no 500 server errors. 

**At least one person needs to approve pull request before merged to master**
